### PR TITLE
Update RSA exception

### PR DESCRIPTION
**Pull request by jbrown-xentity**
Related to https://github.com/GSA/datagov-deploy/issues/2472

### CODE REVIEW

File: .snyk [hunk 1]
```diff
@@ -26,7 +26,8 @@ ignore:
   SNYK-PYTHON-M2CRYPTO-1040426:
     - '*':
         reason: >-
-          No remediation path available. Issue created: 
-          https://github.com/GSA/datagov-deploy/issues/2472
-        expires: 2020-12-23T06:00:00.000Z
+          Vulnerable code functions are not utilized in CKAN code base (RSA decrypt functionality).
+          No fix for M2Crypto (RSA module) at this time.
+          Community believes this to be a medium severity vulnerability.
+        expires: 2020-02-03T06:00:00.000Z
```
Comment (FuhuXia) [inline .snyk:32]: 2021, not 2020

**Review by FuhuXia — CHANGES_REQUESTED** [review]

File: .snyk (hunk 1 above)
Comment (adborden) [reply .snyk:32]:
```suggestion
        expires: 2021-02-03T06:00:00.000Z
```

Comment (adborden) [reply .snyk:32]: Can you clarify how/if we're using the vulnerable code from M2Crypto?

**Review by chris-macdermaid — APPROVED** [review]

**Comment by jbrown-xentity** [comment]
Now defunct, closing.

**Pull request closed without merging**